OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameMenuLogic.cs

```diff
@@ -133,6 +154,8 @@ public class IngameMenuLogic : ChromeLogic
 		bool leaving;
 		bool hideMenu;
 
+		static bool lastGameEditor = false;
```

Comment:
This static flag on the widget seems hacky, i'm not sure I have a better suggeston but it feels like we should be managing this at something higher level that keeps track of global UI/Game state and not within the widget directly.

Reply:
I believe we had long discussions about how to implement this. This was the best one we came up with